### PR TITLE
Add localization support

### PR DESCRIPTION
**Pull request by dyon**
# Description
This PR adds localization support. The active locale can be changed from the app that imports this component library by importing the `i18n.js` file and calling the `changeLanguage(language)` function. A README file has been added to describe this process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+### Change active locale at runtime
+
+The project that's using the agent-ui can dictate which locale to use in our components. Below is an example of how to
+achieve this from an app that's also using `i18next` to handle localization.
+
+```js
+// i18n.js
+
+import i18n from 'i18next';
+import i18nAgentUI from 'cobrowse-agent-ui/dist/i18n';
```
Comment (headlessme) [inline README.md]: Let's expose this as part of the main API:

```
import { i18n } from 'cobrowse-agent-ui'
```

**Review by headlessme — COMMENTED** [review]
Looks good, I think this approach will make things simpler for us as well as most customers.

File: README.md (hunk 1 above)
Comment (dyon) [reply README.md]: Done!

Comment (headlessme) [inline README.md]: Please update here too to avoid confusion!

**Review by headlessme — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dyon) [reply README.md]: Whoops. Done!

**Pull request merged**